src/main/java/org/kainos/ea/utility/DatabaseConnector.java

```diff
@@ -0,0 +1,41 @@
+package org.kainos.ea.utility;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import org.kainos.ea.exceptions.DatabaseConnectionException;
+
+public class DatabaseConnector {
+
+  private static Connection conn;
+
+  public Connection getConnection() throws DatabaseConnectionException, SQLException {
+    String user;
+    String password;
+    String host;
+    String database;
+
+    if (conn != null && !conn.isClosed()) {
+      return conn;
+    }
+
+    try {
+      user            = System.getenv("DB_USERNAME");
+      password        = System.getenv("DB_PASSWORD");
+      host            = System.getenv("DB_HOST");
+      database        = System.getenv("DB_NAME");
```

Comment:
Are we able to use the `DbCredentials` class here in place of this?

Reply:
Yes, good shout, i've updated this now.